README.md

```diff
@@ -324,15 +330,26 @@ def test_run_example_journey(complete_configuration):
     ExampleJourney.update_incoming_scenario_args(complete_configuration)
     ExampleJourney.update_incoming_scenario_args({
         "thresholds": {
-            "{GET}get google": 4000, # 4 second HTTP response threshold
-            "{CUSTOM}my custom trend": 11000 # 11 second custom trend threshold
-            }
-        }) 
+            "get google":
+                {
+                    "type": "get",
+                    "limit": 4000  # 4 second HTTP response threshold
+                },
+            "my custom trend":
+                {
+                    "type": "custom",
+                    "limit": 11000  # 11 second custom trend threshold
+                }
+        }
+    })
     
     locust_env = Grasshopper.launch_test(ExampleJourney, **complete_configuration)
     return locust_env
 ```
 
+Thresholds can also be defined for individual YAML scenarios. Refer to the `thresholds` 
```

Comment:
did you also want to mention setting thresholds in the defaults on a journey class as well?